examples/tutorials/Scanpy.ipynb

```diff
@@ -21,37 +21,25 @@
     "\n",
```

Comment:
<p>Let's go one step further; can we train a model with this data and DeepChem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='133'/>